xc/device.ml

```diff
@@ -2178,11 +2178,10 @@ module Dm_Common = struct
           ; sprintf "media=%s" (string_of_media media)
           ] @ (format_of_media media file))
     ; "-device"; String.concat ","
-        ([ "nvme"
-         ; sprintf "serial=%s" id
+        ([ "nvme-ns"
          ; sprintf "drive=%s" id
-         ; sprintf "addr=%d" 7
-           (* 4 and 5 are NICs, and we can only have two, 6 is platform *)
+         ; sprintf "bus=nvme%d" 0
```

Comment:
If we only ever have 1 NVME bus can we avoid using %d?

Reply:
Yes.